tools/build/bin/betools.js

```diff
@@ -40,6 +40,9 @@ yargs.strict(true)
         "excludeGlob": {
           describe: "Specify a directory, filename, or pattern to be excluded"
         },
+        "testExcludeGlob": {
```

Comment:
Why do you need a new `testExcludeGlob` rather than using the existing `excludeGlob`?

Reply:
Exclude for `**/*test*/**/*` is hardcoded and `excludeGlob` just appends to it. So to resolve my issue `excludeGlob` should override the default exclude `**/node_modules/**/*,**/*test*/**/*`. Of course that would be breaking change if some package uses `excludeGlob` and relies on exclude for `*test*` and `node_modules` to be added be docs script.

So I was trying to add away to override that hardcoded glob for tests without introducing any required changes for other packages using `docs` script. If we can assume that this script is used only by packages in `itwinjs-core` repo (and `appui`) I guess I can remove hardcoded exclude for tests and update docs script in package.json files to pass `excludeGlob` for test files.

Reply:
ah okay, missed that we default to excluding that test folder.